src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -8056,7 +7986,7 @@ components:
         type: integer
         default: 0
     LoadTag:
-      name: loadtag
+      name: loadTag
```

Comment:
This one might have functional changes in the code? Did the codegen change the variable name?

Reply:
I think in this case this schema entry wasn't used at all before, and this changes it to match the uses that it's replacing above.

Reply:
Yes, code gen changed the variable name